.cirrus.yml

```diff
@@ -234,6 +229,7 @@ promote_task:
   <<: *MAVEN_CACHE
   env:
     <<: *PROMOTE_SECRETS
+    # TODO: Remove once cirrus_promote_maven doesn't call burgr_notify anymore
     #artifacts that will have downloadable links in burgr
     ARTIFACTS: org.sonarsource.php:sonar-php-plugin:jar
```

Comment:
These two lines can be removed as well, `ARTIFACTS` is used by burgr script
https://github.com/SonarSource/re-ci-images/blob/1b6d467bf00306e3d81655327473e53ce92c9583/docker/bin/src/burgr-notify-promotion#L6